finetuner/tailor/pytorch/projection_head.py

```diff
@@ -0,0 +1,30 @@
+import torch.nn as nn
+
+
+class ProjectionHead(nn.Module):
+    """Projection head used internally for self-supervised training.
+    It is (by default) a simple 3-layer MLP to be attached on top of embedding model only for training purpose.
+    After training, it should be cut-out from the embedding model.
+    """
+
+    EPSILON = 1e-5
```

Comment:
Any reason we are specifying this manually instead of leaving it at default?

Reply:
different frameworks might have different values, i want to set them as the same @tadejsv

Reply:
sounds good